.github/workflows/gradle_localstack_postgres_verify.yml

```diff
@@ -0,0 +1,92 @@
+name: Run gradle checks with localstack and postgres instances
+on:
+  workflow_call:
+    inputs:
+      java-version:
+        type: string
+        required: false
+        default: '21'
+      java-options:
+        type: string
+        default: ''
+        required: false
+      postgres-tag:
+        type: string
+        required: true
+      postgres-password:
+        type: string
+        default: 'dev'
+        required: false
+      postgres-username:
+        type: string
+        default: 'root'
+        required: true
+      postgres-db:
+        type: string
+        default: 'postgres'
+        required: false
+      gradle-command:
+        type: string
+        default: './gradlew check'
+        required: false
+      services:
+        type: string
+        default: 'sqs,sns'
+        required: false
+      localstack-tag:
+        type: string
+        default: 'latest'
+        required: false
+permissions:
+  contents: read
+jobs:
+  gradle-localstack-postgres-verify:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        # Docker Hub PostgreSQL image
+        image: postgres:${{ inputs.postgres-tag }}
+        # Provide the configuration for postgres
+        env:
+          POSTGRES_USER: ${{ inputs.postgres-username }}
+          POSTGRES_PASSWORD: ${{ inputs.postgres-password }}
+          POSTGRES_DB: ${{ inputs.postgres-db}}
+        options: >-
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+      localstack:
```

Comment:
How do you know when localstack is ready to serve traffic?

Reply:
I thought that was odd too, but that is the way that https://github.com/ministryofjustice/hmpps-github-actions/blob/main/templates/workflows/kotlin_localstack.yml is written.  I'll fix.